File: cs50x/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

//Prints output piramid
void Output(int Input)
{
    for (int a = 1; a < Input + 1; a++)
    {
        for (int c = Input - a; c > 0; c--)
        {
            printf(" ");
        }
        for (int b = 1; b < a + 1; b++)
        {
            printf("#");
        }
        printf("  ");
        for (int b = 1; b < a + 1; b++)
        {
            printf("#");
        }
        printf("\n");
    }
}

//main function
int main(void)
{
    int Input = 0;
    do
    {
        Input = get_int("Height: ");

    }
    while (Input < 1 || Input > 8);
    Output(Input);
}

File: cs50x/pset1/mario.py
from cs50 import get_int

# Prints output piramid.


def Output(Input):
    for i in range(1, Input + 1):
        for j in range(Input - i, 0, -1):
            print(" ", end="")
        for j in range(1, i + 1):
            print("#", end="")
        print("  ", end="")
        for j in range(1, i + 1):
            print("#", end="")
        print("")

# Main function.


def main():
    Input = int(0)
    while (Input < 1 or Input > 8):
        Input = get_int("Height: ")
    Output(Input)


if __name__ == "__main__":
    main()